app/assets/stylesheets/application.scss

```diff
@@ -1,3 +1,4 @@
 @import "lib/*";
 @import "vendor/vendor";
 @import "components/*";
+@import "helper/bootstrap_overrides";
```

Comment:
For consistency, we should probably either use globbing or not use globbing if you know what I mean